libs/components/router/src/lib/modules/href/href.directive.ts

```diff
@@ -34,12 +34,16 @@ export class SkyHrefDirective {
   @Input()
   public set skyHref(skyHref: string | string[]) {
```

Comment:
This input's value can technically be `undefined`.

Reply:
Fixed